modules/client/src/multisig/interfaces.ts

```diff
@@ -204,4 +204,3 @@ export type SubgraphMultisigVotingSettings = {
   minApprovals: number;
   onlyListed: boolean;
 };
-
```

Comment:
```suggestion
};

```

Reply:
idk why it has the wron formatting in gh in mylocal seems ok with an empty line at the end

Reply:
yes but on my machine the empty line is still there, idk why is it removed on gh

Reply:
Very weird...
<img width="769" alt="image" src="https://github.com/aragon/sdk/assets/4456749/e047fb97-a0a8-4415-8ea2-64fe4e163f62">

Reply:
well, i'll commit it and hopefully it fixes